libass/ass_utils.h

```diff
@@ -104,7 +104,11 @@ int numpad2align(int val);
 unsigned ass_utf8_get_char(char **str);
 unsigned ass_utf8_put_char(char *dest, uint32_t ch);
 void ass_utf16be_to_utf8(char *dst, size_t dst_size, uint8_t *src, size_t src_size);
-void ass_msg(ASS_Library *priv, int lvl, const char *fmt, ...);
+void ass_msg(ASS_Library *priv, int lvl, const char *fmt, ...)
+#ifdef __GNUC__
+    __attribute__ ((format (printf, 3, 4)))
+#endif
+;
```

Comment:
Nit: can’t this go before `void ass_msg` to avoid leaving the semicolon all alone on a line?

Reply:
Seems like it. Both gcc's and clang's doc list examples of function attributes *before* the function and it builds fine (and warns about mistakes) with both gcc and clang for me. Moved it.